src/transformers/utils/doc.py

```diff
@@ -172,28 +186,31 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> from transformers import {processor_class}, {model_class}
     >>> import torch
 
-    >>> torch.manual_seed(0)  # doctest: +IGNORE_RESULT
-
     >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
     >>> model = {model_class}.from_pretrained("{checkpoint}")
 
     >>> question, text = "Who was Jim Henson?", "Jim Henson was a nice puppet"
```

Comment:
I have to admit that I don't know who Jim Henson is before google 😭. 
Personally, I would like to have examples that I know **very clearly** the model gives correct answer.
But no a big deal.